packages/core/src/controllers/verify.ts

```diff
@@ -1,76 +1,130 @@
 import { generateChildLogger, getLoggerContext, Logger } from "@walletconnect/logger";
 import { IVerify } from "@walletconnect/types";
-import { isBrowser, isNode, isReactNative } from "@walletconnect/utils";
+import {
+  getCryptoKeyFromKeyData,
+  isBrowser,
+  isNode,
+  P256KeyDataType,
+  verifyP256Jwt,
+} from "@walletconnect/utils";
 import { FIVE_SECONDS, ONE_SECOND, toMiliseconds } from "@walletconnect/time";
+import { getDocument } from "@walletconnect/window-getters";
 
-import { TRUSTED_VERIFY_URLS, VERIFY_CONTEXT, VERIFY_SERVER } from "../constants";
+import { TRUSTED_VERIFY_URLS, VERIFY_CONTEXT, VERIFY_SERVER, VERIFY_SERVER_V2 } from "../constants";
+import { IKeyValueStorage } from "@walletconnect/keyvaluestorage";
 
+type jwk = {
```

Comment:
nit type should be uppercased
```suggestion
type Jwk = {
```